enable-vcp-image/enable-vcp-scripts/daemonset_pod.sh

```diff
@@ -3,8 +3,16 @@ source $(dirname "$0")/common_func.sh
 source $(dirname "$0")/exit_codes.sh
 
 [ -z "$POD_NAME" ] && { echo "[ERROR] POD_NAME is not set"; exit $ERROR_POD_ENV_VALIDATION; }
-[ -z "$NODE_NAME" ] && { echo "[ERROR] NODE_NAME is not set"; exit $ERROR_POD_ENV_VALIDATION; }
-[ -z "$POD_ROLE" ] && { echo "[ERROR] POD_ROLE is not set"; exit $ERROR_POD_ENV_VALIDATION; }
+init_VcpConigStatus "$POD_NAME"
+
+PHASE=$DAEMONSET_SCRIPT_PHASE1
+update_VcpConigStatus "$POD_NAME" "$PHASE" "RUNNING" ""
+
+ERROR_MSG="NODE_NAME is not set"
+[ -z "$NODE_NAME" ] && { echo "[ERROR] ${ERROR_MSG}"; PHASE_STATUS="FAILED"; update_VcpConigStatus "$POD_NAME" "$PHASE" "FAILED" "$ERROR_MSG"; exit $ERROR_POD_ENV_VALIDATION; }
+
+ERROR_MSG="POD_ROLE NODE_NAME is not set"
```

Comment:
Remove NODE_NAME from this ERROR_MSG?
If NODE_NAME is not set, it's already error out above.